### ISSUE TITLE
Improve model

### ISSUE THREAD

**Issue by sftwre** [leviv/TwitterStyleTransfer#10]
Training losses do not converge. Generated text indicates poor training.

**Comment by sftwre** [comment]
Add attention matrix to decoder input to penalize repeating text.

**Comment by sftwre** [comment]
More tricks we can use to improve generation:

- [x] Clip gradients
- [ ] Weight decay
- [ ] Beam search 
- [ ] Anneal learning rate if validation loss does not improve
- [x] Reduce vocabulary
- [ ] Character level tokenization
